babel.config.js

```diff
@@ -34,8 +34,8 @@ const config = api => {
          * package root, like 'src/classify'.
          */
         development: {
-            // Ignore __mocks__, __stories__, etc.
-            ignore: [/\/__\w+__\//],
```

Comment:
This same pattern is in `packages/pwa-buildpack/.babelrc.js`. I don't see a use case for pwa-buildpack to have any ui components, but if you wanted to future proof that package, change it there too.